src/slide-content/slides.ts

```diff
@@ -26,7 +26,39 @@ export interface ImageSlideContent {
   imageSrc: string;
 }
 
+export interface SpeakersSlideContent {
+  type: "speakers";
+  title?: string;
+  speakers: Speaker[];
+}
+
+export interface Speaker {
+  name: string;
+  twitter: string;
+  bio: string;
+  pictureUrl: string;
+}
+
 export const slides: SlideContent[] = [
+  {
+    type: "speakers",
+    speakers: [
+      {
+        name: "Aurélien Loyer",
+        twitter: "@AurelienLoyer",
+        bio: "Software Engineering Leader @QIMA",
+        pictureUrl:
+          "https://s3.amazonaws.com/media-p.slid.es/uploads/637110/images/8938477/pasted-from-clipboard.png",
```

Comment:
We should host the images ourself in the `public` folder for the offline mode :octocat: